### PR TITLE
Update heaps.py

### LINKED ISSUE #107: Adding `sort` method to heaps

### PR DESCRIPTION
**Pull request by ramyaperumandla24**
#### References to other Issues or PRs or Relevant literature

Fixes #107

### CODE REVIEW

**Comment by czgdp1807** [comment]
Which issue is fixed by this PR?

**Comment by ramyaperumandla24** [comment]
Adding 'sort' methods to heaps.

**Comment by ramyaperumandla24** [comment]
Issue number 107

**Comment by czgdp1807** [comment]
I think it would be better to have a `sort` method in `DHeap` which will return all the elements of the heap in sorted order. See, https://github.com/codezonediitj/pydatastructs/issues/107#issuecomment-593580517 as well and adjust your code accordingly.

**Comment by ramyaperumandla24** [comment]
i have changed the code i.e., placed that sort method in Dheap and created pullrequest. can you tell me what is wrong in that?

File: pydatastructs/trees/heaps.py [hunk 1]
```diff
@@ -18,6 +18,44 @@ class Heap(object):
 
 
 class DHeap(Heap):
+     def heapsort( aList ):
```
Comment (czgdp1807) [inline pydatastructs/trees/heaps.py]: The method should have been placed after all the methods. Do not place methods before the doc strings.

Comment (czgdp1807) [reply pydatastructs/trees/heaps.py]:
```suggestion
     def heapsort(self, aList):
```

**Comment by czgdp1807** [comment]
Closing in reference to https://github.com/codezonediitj/pydatastructs/wiki/Pull-Request-Policy#when-to-close-a-pr

**Pull request closed without merging**